.github/workflows/review.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         run: yarn format:ci
 
       - name: Review dependencies
-        run: yarn depcheck
+        run: yarn depcheck --ignore-path="examples"
```

Comment:
what's the issue here? 
wouldn't it make more sense just to update the SDK version there, so we have it always up to date?

Reply:
SDK is using ethers v5 which is completely differnet from v6, and in the example we have the newest v6
We can make changes to the RPCh SDK if you want, but I would not do them in this PR